### DISCUSSION TITLE
VB.net support

### DISCUSSION THREAD

**Discussion by pamidur** [pamidur/aspect-injector#183] [General]
We want to make sure that VB.net is properly supported. 
This discussion is dedicated to this problem

**Comment by pamidur** [comment]
@Meigs2, how the 2.7.3-pre1 works for you? I don't use VB that much to validate that it works fine

**Comment by Meigs2** [comment]
@pamidur I've been slightly busy as of late, sorry about that! I will push my fork of the VB.NET test suite I converted and touched up to GitHub later today and reply in here when it's up.

IIRC, everything is passing except from one test since the last pre-release.
